### ISSUE TITLE
Spurious "multipe instances" warning

### ISSUE THREAD

**Issue by gdementen** [ZoomerAnalytics/xlwings#454]
As soon as I try to open a workbook which is already open (once!) I get a warning:

'This Workbook is open in multiple instances.'
'The connection was made with the one that was last active.'

FWIW, I am on Windows 10, using Office 2013.

This is annoying... I tried to trace the problem down but hit a brick wall as I don't know enough of the internals to understand exactly what happens. Here are a few tidbits I figured. Hope it helps.

I just figured that get_duplicate_fullnames() always returns all open files (whether they are opened in one instance or several), that get_xl_apps() returns me **one more** xl_app than the number of workbooks I opened and that get_all_open_xl_workbooks() seems to return all open workbooks **for each xl_app**.

So for example, if I have 3 documents open:
* get_duplicate_fullnames() returns all 3 documents
* get_xl_apps() returns 4 instances.
* each call to get_all_open_xl_workbooks(xl_app) returns all 3 documents, so the
open_xl_workbooks list at the end of get_duplicate_fullnames contains 12 elements, which obviously contains duplicates...

**Comment by fzumstein** [comment]
What do you do exactly? 

Open the workbooks manually in two instances? then trying to connect with `Workbook(path)`?

I guess the issue is that xlwings doesn't allow you to specify the Excel instance when making a connection via `Workbook`, but that's currently being worked on (`bluesky` branch).

**Comment by gdementen** [comment]
For my tests, I had manually opened 3 workbooks (via double clicking the file, or via file->open in Excel, this does not seems to make a difference), then try to open one of those 3 workbooks via xlwings.Workbook(path)

It is possible to reproduce this by using only xlwings:
```python
>>> import xlwings as xw
>>> import os

>>> # no warning
>>> wb1 = xw.Workbook(os.path.abspath('test1.xlsx'))
>>> # no warning
>>> wb2 = xw.Workbook(os.path.abspath('test1.xlsx'))
>>> # no warning
>>> wb3 = xw.Workbook(os.path.abspath('test1.xlsx'))
>>> # no warning
>>> wb4 = xw.Workbook(os.path.abspath('test2.xlsx'))
>>> # get the warning
>>> wb5 = xw.Workbook(os.path.abspath('test2.xlsx'))
c:\users\gdm\devel\xlwings\xlwings\_xlwindows.py:170: UserWarning: This Workbook is open in multiple instances.The connection was made with the one that was last active.
  warn('This Workbook is open in multiple instances.'
```
It seems to be that if you have more than one workbook open and you try to open one of them, you get the warning because all instances of excel seem to report having all the workbook as open.

**Comment by fzumstein** [comment]
Ah I see, so you have each workbook open just 1x? i.e. test1 in instance 1 and test2 in instance 2 etc.?

**Comment by gdementen** [comment]
Yes. Maybe the example above wasn't clear. It was just meant to demonstrate that with one file open I have no warning, but with two I get a warning, see below. Btw, this whole thing is with xlwings 0.7.1.

```python
>>> # I close all instances of Excel (and kill any remaining one in Task Manager) before doing the test.
>>> import xlwings as xw
>>> xw.__version__
'0.7.1'
>>> import os

>>> # no warning
>>> wb1 = xw.Workbook(os.path.abspath('test1.xlsx'))
>>> # no warning
>>> wb2 = xw.Workbook(os.path.abspath('test2.xlsx'))
>>> # get a warning if I try to open either test1 or test2 (assuming they are still open)
>>> wb3 = xw.Workbook(os.path.abspath('test2.xlsx'))
c:\users\gdm\devel\xlwings\xlwings\_xlwindows.py:170: UserWarning: This Workbook is open in multiple instances.The connection was made with the one that was last active.
  warn('This Workbook is open in multiple instances.'

**Comment by fzumstein** [comment]
Ok all clear now - I ll look into it.

**Comment by fzumstein** [comment]
Looks like the issue is caused by the change they introduced with Excel 2013 to open each workbook in an own window.

**Comment by fzumstein** [comment]
fix: fc2c571563f4f0ca15e6ec685be0c139f989af96

**Comment by gdementen** [comment]
Thanks.
